ibis/backends/duckdb/tests/test_client.py

```diff
@@ -292,15 +293,17 @@ def test_list_tables_schema_warning_refactor(con):
         "diamonds",
         "functional_alltypes",
         "win",
-    }.issubset(con.list_tables())
+    }.issubset(con.tables)
 
     icecream_table = ["ice_cream"]
 
     with pytest.warns(FutureWarning):
+        # I'M HAVING AN ISSUE HERE THAT THE TEST ITSELF RUNS BUT WHEN I RUN
```

Comment:
I'm not sure what's happening here, I think it might be related to the caching. when I run this test isolated it passes, but when I run the whole `test_client.py` the LHS ends up with more tables that I'm not sure where they are coming from.